commands/general/stats.js

```diff
@@ -4,7 +4,7 @@ const moment = require('moment')
 
 module.exports.load = (client) => {
   client.commands['stats'] = {
-    run (message) {
+    async run (message) {
```

Comment:
See #35 also.

Reply:
Yep, probably gonna make a few changes to that one and check it in as well. Just want to un-callback-ify as much of the code as possible before I continue on the replat.